roles/users/tasks/main.yml

```diff
@@ -1,12 +1,13 @@
 ---
-- name: Setup web user
-  user: name="{{ web_user }}"
-        group="{{ web_group }}"
+- name: Setup users
+  user: name="{{ item.name }}"
+        group="{{ item.groups[0] }}"
+        groups="{{ item.groups | join(',') }}"
+        password="{{ sudoer_passwords[ item.name ] | default(None) }}"
```

Comment:
Was copy-pasted from [`sudoers` role](https://github.com/roots/bedrock-ansible/blob/ef70af82afad5c48f921a89b19d5a17022ea6b7d/roles/sudoers/tasks/main.yml#L16), but I'll leave that one alone instead of changing it too.